application_store/_helpers/application.py

```diff
@@ -64,28 +69,49 @@ def send_submit_notification(
                 return  # we don't send emails for a failure
 
             case Decision.PASS:
-                notify_template = Config.NOTIFY_TEMPLATE_EOI_PASS
+                get_notification_service().send_eoi_pass_email(
+                    email_address=account.email,
+                    contact_name=full_name.title() if full_name else None,
+                    language=language,
+                    fund_id=fund_id,
+                    fund_name=fund_name,
+                    application_reference=application_reference,
+                    submission_date=submission_date,
+                    round_name=round_name,
+                    questions=questions,
+                    contact_help_email=round_data.contact_email,
+                )
 
             case Decision.PASS_WITH_CAVEATS:
-                notify_template = Config.NOTIFY_TEMPLATE_EOI_PASS_W_CAVEATS
-                contents[NotifyConstants.APPLICATION_CAVEATS] = eoi_results["caveats"]
+                get_notification_service().send_eoi_pass_with_caveats_email(
```

Comment:
Do we want to explicitly handle errors here? This might be the only thing relevant to the short term change. In the places this is swapped out for are we saying that any raised exceptions should stop the process.

Do we want to put in any kind of monitoring/ logging to review failure modes and frequency?

I'm guessing most of the time a failed email should void the transaction/ process but I like the idea of being able to swap to this and see how often emails might fail

In the current world of sending to SQS there's very little that could fail at the time of submitting the SQS message, with any downstream email failures being stumbled on the notification service - that makes mapping how the error handling _should_ work quite strange